src/search.ts

```diff
@@ -71,8 +72,32 @@ function nakDefinitionSearch(document: vscode.TextDocument, pos: vscode.Position
         let word = document.getText(range);
         let pattern = `(let|const|var|function|class)\\s+${word}|${word}\\s*:`
         let cmd = `node ${module} --ackmate -G "*${extname(document.fileName)}" -d "*node_modules*" "${pattern}" ${vscode.workspace.rootPath}`;
-        const nak = exec(cmd, (err, stdout, stderr) => {
+        
+        // Settings: TODO: Expose below extension settings to users (possibly pattern for advanced users too?)
+
```

Comment:
Better not - once we have https://github.com/microsoft/vscode/issues/73524 this extension can re-use the VS Code search infrastructure.